plugins/ubiquiti/unifi_api

```diff
@@ -0,0 +1,1138 @@
+    # Show number of clients connected to each device
+    env.enable_detail_clients_device yes
+    # Show detailed graphs for each device
+
+    env.enable_clients_type yes
+    # Show number of clients connected to each network type
+    env.enable_detail_clients_type yes
+    # Show detailed graphs for each client type
+    env.show_authorized_clients_type yes
+    # Show unauthorized / authorized client list
+    # if you are not using the guest portal, this is useless
+
+    env.enable_xfer_port yes
+    # Show transfer statistics on switch ports
+    env.enable_detail_xfer_port yes
+    # Show detailed graphs per switch port
+    env.hide_empty_xfer_port yes
+    # Hide ports that have no link
+
+    env.enable_xfer_device yes
+    # Show transfer statistics per device
+    env.enable_detail_xfer_device yes
+    # Show detailed graphs for each device
+
+    env.enable_xfer_network yes
+    # Show transfer statistics per named network
+    env.enable_detail_xfer_network yes
+    # Show detailed graphs for each named network
+
+    env.enable_xfer_radio yes
+    # Show transfer statistics per radio
+    env.enable_detail_xfer_radio yes
+    # Show detailed graphs for each radio
+
+
+=head1 CAPABILITIES
+
+This plugin requires multigraph.
+
+This plugin supports L<DIRTYCONFIG|http://guide.munin-monitoring.org/en/latest/plugin/protocol-dirtyconfig.html>
+
+=head1 DEPENDENCIES
+
+=over
+
+=item WWW::Curl::Easy
+
+Perl extension interface for libcurl
+
+=item JSON
+
+ JSON (JavaScript Object Notation) encoder/decoder
+
+=back
+
+=head1 PERFORMANCE CONCERNS
+
+The main performance concern on this is the huge number of graphs that may be
+generated.  Using the cron version of munin-graph may hurt a lot.
+
+A bit of a case study:
+
+                 | My Site  | UBNT Demo
+---------------------------------------
+Devices          |       8  |     126
+AP's             |       4  |     118
+24xSwitch        |       1  |       5
+8xSwitch         |       2  |       2
+Output Bytes     |  64,262  | 431,434
+Output Lines     |   1,761  |  14,586
+Output Graphs    |      77  |     530
+
+So, just note that the growth in the amount of graphed date can be extream.
+
+
+=head1 LICENSE
+
+Copyright (C) 2018 J.T.Sage (jtsage@gmail.com)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see L<http://www.gnu.org/licenses/>.
+
+=head1 VERSION
+
+Most recent version available from 
+
+L<https://github.com/jtsage/unifi-munin-plugins>
```

Comment:
Do you plan to maintain of the plugin(s) in your repository in the long term?
In this case I would suggest, that we do not try to maintain a code copy here in this munin contrib repository, since this will increase the maintenance burden with no real benefit.

Of course, we are talking about free software - thus there is no way to stop anyone from maintaining code copies.
Please feel free to follow your own judgement when deciding for a primary location of this code. In any case it would probably be misleading to distribute code that refers to an independent location (with content that may/will vary over time).

Reply:
Yeah, this repo should be it's home - I tend to develop with a github repo, if for no other reason than to remind myself of what I did - these actually started as PHP scripts, with a 3rd party API include - not the best method, but, should I need to use that for something else, it's in the archive.

I will remove the reference.